models/questions.py

```diff
@@ -12,7 +12,6 @@
                 Field('htmlsrc', type='text'),
                 Field('practice', type='boolean'),
                 Field('autograde', type='string'),
-                Field('practice', type='boolean'),
```

Comment:
this field is needed for the new spaced practice feature

Reply:
But it is needed twice? (It also appears in line 13 above).

Reply:
No, we don't need it twice!  I didn't see that it was duplicated.  Amazing that it doesn't cause a database error.